atima/README.md

```diff
@@ -0,0 +1,32 @@
+# AtimaWrapper for R3BRoot
+
+You can get energyloss etc. from Atima inside R3BRoot with only a few lines of code.
+The following line, e.g. will give you the Energyloss of a proton beam @ 400 MeV in a 50mm liquid hydrogen target:
+
+```c++
+    R3BAtima::Calculate_mm(1, 1, 400, R3BAtima::LiqH, 50).ELoss_AMeV;
+``` 
+
+You can define Materials using R3BAtimaTargetMaterial and R3BAtimaMaterialCompound or use predefined Materials inside R3BAtima.h.
+Please add commonly used materials which are not defined there yet!
+
+Since the computation takes a small amount of time, you can store the result in a chosen range in a so called map in order to increase the speed in frequent computations.
+
+The following lines will create a map which can be read out whithin the chosen range. The results will be interpolated from the precalculated points.
+
+```c++
+    // Energie from 100 AMeV to 200 AMeV with 10 steps
+    // Distance from 0mm to 50mm with 20 steps
+    auto header = R3BAtimaMapHeader(1, 1, R3BAtima::LiqH, {100,200,10}, {0,50,20}); 
+    auto atimamap = R3BAtimaMap(header);
+    atimamap.GetResult(150,4.9).ELoss_AMeV; // ELoss with 150 AMeV and 4.9mm
```

Comment:
I don't really like the Map/MapHeader structure and naming. The map cannot live without the header, and the header alone is useless. What about:
```C++
auto atimarange = R3BAtimaRange(1, 1, {100,200,10}, R3BAtima::LH2, {0,50,20});
atimarange.GetResult(150, 4.9).ELoss_AMeV;
atimarange.Save("atima.cache");
````
Notice also the parameter order change to be equivalent to R3BAtima.

To load 
```C++
auto atimarange = R3BAtimaRange::Load("atima.cache");
```

Reply:
I decided for the constructor with the second argument so you do not have to bother with creating the file and loading etc inside. You just enter the line and it will load or create the file.

Reply:
On the Map/Header issue:
When I started this I went this way because I was hopeing that this would make it easier to understand for students.
I.e. you define what you want to look at in one line, and than use it in the second.
But after finishing I barely see any advantage.